pkg/services/folder/folderimpl/dashboard_folder_store.go

```diff
@@ -82,5 +82,7 @@ func (d *DashboardFolderStoreImpl) GetFolderByUID(ctx context.Context, orgID int
 	if err != nil {
 		return nil, err
 	}
-	return dashboards.FromDashboard(&dashboard), nil
+
+	res := dashboards.FromDashboard(&dashboard)
```

Comment:
Is there a reason you added the `res` variable here? It's not a problem, but it looks like unnecessary memory allocation without a benefit.

Reply:
correct, we don't need that: initially I had the caching in this store and it remained there by mistake when I removed it.